code/modules/mob/living/simple_animal/friendly/fox.dm

```diff
@@ -6,23 +6,23 @@
 	icon_state = "fox"
 	icon_living = "fox"
 	icon_dead = "fox_dead"
-	speak = list("Ack-Ack","Ack-Ack-Ack-Ackawoooo","Geckers","Awoo","Tchoff")
-	speak_emote = list("geckers", "barks")
-	emote_hear = list("howls.","barks.")
+	speak = list("AAAAAAAAAAAAAAAAAAAA","Yip","Hehehehehe")
+	speak_emote = list("screamss")
+	emote_hear = list("yips.","screeches.")
 	emote_see = list("shakes its head.", "shivers.")
 	speak_chance = 1
 	turns_per_move = 5
 	see_in_dark = 6
-	butcher_results = list(/obj/item/reagent_containers/food/snacks/meat/slab = 3)
+	butcher_results = list(/obj/item/reagent_containers/food/snacks/meat/slab = 1) //3 -> 1, foxes aren't Big
 	response_help = "pets"
 	response_disarm = "gently pushes aside"
 	response_harm = "kicks"
 	attack_vis_effect = ATTACK_EFFECT_BITE
 	gold_core_spawnable = FRIENDLY_SPAWN
 	can_be_held = TRUE
 	do_footstep = TRUE
-	wuv_happy = "yaps happily!"
-	wuv_angy = "growls!"
+	wuv_happy = "screams happily!"
+	wuv_angy = "screams angerly!"
```

Comment:
angrily

Reply:
it was a typo ye, not intentional, will fix soon